portfolio/src/main/webapp/script.js

```diff
@@ -29,6 +29,38 @@ function addRandomShow() {
 
 async function getName() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('hello-container').innerText = quote;
+  const name = await response.text();
+  document.getElementById('hello-container').innerText = name;
+}
+async function getDataStats() {
+  /*fetch('/data').then(response => response.json()).then((stats) => {
+    // stats is an object, not a string, so we have to
+     //reference its fields to create HTML content
+    const statsListElement = document.getElementById('data-stats-container');
+    statsListElement.innerHTML = '';
+    statsListElement.appendChild(createListElement(stats));
+  }); */
+  const response = await fetch('/data');
+  const colleges = await response.text();
```

Comment:
If you use `response.json()` instead of `response.text()`, it will parse the JSON data for you.